apps/docs/app/root/layout/SearchDocs/SearchDocsInput.tsx

```diff
@@ -0,0 +1,121 @@
+import { Combobox, createListCollection } from "@ark-ui/react/combobox";
+import { SearchOutline24Icon } from "@vygruppen/spor-icon-react";
+import { SearchInput, Text } from "@vygruppen/spor-react";
+import { useState } from "react";
+import { useMenu } from "~/utils/useMenu";
+import { matchSorter } from "match-sorter";
+import { Link, useNavigate } from "react-router-dom";
+import { chakra, For } from "@chakra-ui/react";
+
+const useSearchableItems = () => {
+  const menu = useMenu("side-menu");
+
+  if (!menu?.menuItems) return [];
+
+  return menu.menuItems
+    .filter(({ _type, subItems }) => _type !== "divider" && subItems)
+    .flatMap(
+      ({ title, subItems }) =>
+        subItems?.map((subItem) => ({
+          ...subItem,
+          categoryTitle: title ?? "",
+          value: subItem.url,
+        })) ?? [],
+    );
+};
+
+const ComboboxItem = chakra(
+  Combobox.Item,
+  {
+    base: {
+      display: "flex",
+      flexDirection: "column",
+      borderRadius: "xs",
+      gap: "1",
+      px: "4",
+      py: "3",
```

Comment:
for å være mer lesbar vi brukt fra starten full syntaks paddingX og paddingY da er mer konsekvent med resten av codebase